classes/Validate.php

```diff
@@ -334,6 +335,27 @@ public static function isNumericIsoCode($iso_code)
         return preg_match(NumericIsoCode::PATTERN, $iso_code);
     }
 
+    /**
+     * Checks if the transformation type is one of the allowed ones.
+     *
+     * @see PatternTransformer
+     *
+     * @param string $transformation
+     *
+     * @return bool
+     */
+    public static function isCurrencyPatternTransformation(string $transformation)
+    {
+        $allowedTransformations = [
```

Comment:
Can be a const in `PatternTransformer` like `PatternTransformer::AVAILABLE_TRANSFORMATIONS`

Reply:
Oh wait! This is the Validate class 😅